x-pack/auditbeat/processors/sessionmd/config.go

```diff
@@ -6,15 +6,19 @@
 
 package sessionmd
 
+import "time"
+
 // Config for add_session_metadata processor.
 type config struct {
-	Backend  string `config:"backend"`
-	PIDField string `config:"pid_field"`
+	Backend        string        `config:"backend"`
+	PIDField       string        `config:"pid_field"`
+	DBReaperPeriod time.Duration `config:"db_reaper_config"`
```

Comment:
```suggestion
	DBReaperPeriod time.Duration `config:"db_reaper_period"`
```
🤷 Seems like these should match